server/validate/email/validate.go

```diff
@@ -434,6 +440,64 @@ func (v *validator) Remove(user t.Uid, value string) error {
 	return store.Users.DelCred(user, validatorName, value)
 }
 
+// SendMail replacement
+//
+func (v *validator) Send_Mail(rcpt []string, msg []byte) error {
```

Comment:
1. The function does not need to be exported, please use lowercase first letter.
2. Use of underscores in function naming is uncommon in Go. Please use a function name in camelCase.

Reply:
rename `Send_Mail` to `sendMail`